src/QuadForm/Lattices.jl

```diff
@@ -1181,34 +1181,49 @@ end
 
 function maximal_sublattices(L::AbsLat, p; use_auto::Bool = false,
                                            callback = false, max = inf)
-  @req base_ring(L) == order(p) "asdsd"
```

Comment:
Is there a problem with "asdsd"? :D

Reply:
Not at all actually, I just put a clearer error message.. what is the meaning of "asdsd" by the way ? :)

Reply:
Just joking. I was too lazy to put a proper error message there.